src/doc/partials/type.hbs

```diff
@@ -83,9 +87,12 @@
 <h3>Title</h3>
 <div class="win-type-title">...</div>
 
-<h4>Subtitle</h4>
+<p class="win-type-subtitle">Subtitle</p>
```

Comment:
ditto here too.  think about a bootstrap developer coming from http://getbootstrap.com/css/#type, why change all the class names for the same functionality?  Additionally, if we don't change the markup given these new names, how will it look?